ReactiveCocoa/AppKit/ActionProxy.swift

```diff
@@ -2,7 +2,7 @@ import AppKit
 import ReactiveSwift
 import enum Result.NoError
 
-internal final class ActionProxy<Owner: AnyObject> {
+internal final class ActionProxy<Owner: AnyObject>: NSObject {
```

Comment:
The `NSObject` is added for KVO interoperability tests to pass. I expect this to be removed by supporting native Swift object in the KVO API.